.eslintrc.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+  },
+  extends: [
+    'airbnb-base',
+  ],
+  globals: {
+    Atomics: 'readonly',
+    SharedArrayBuffer: 'readonly',
+  },
+  parserOptions: {
+    ecmaVersion: 2018,
+    sourceType: 'module',
+  },
+  rules: {
+    "import/extensions": ["error",  "always", {ignorePackages: true} ],
+    "linebreak-style": ['error', 'windows']
```

Comment:
Some of you lines are too long, could you please add the following rule here: `"max-len":` [2, 120, 4, {"ignoreUrls": true}]`

It uses to set up the maximum length to 120 symbols